client/header/activity-panel/index.js

```diff
@@ -185,34 +206,29 @@ export class ActivityPanel extends Component {
 			isPerformingSetupTask && {
 				name: 'help',
 				title: __( 'Help', 'woocommerce-admin' ),
-				icon: (
-					<i className="material-icons-outlined">
-						support
-					</i>
-				),
+				icon: <i className="material-icons-outlined">support</i>,
 			},
 		].filter( Boolean );
 	}
 
 	getPanelContent( tab ) {
+		const { hasUnreadOrders, query, hasUnapprovedReviews } = this.props;
+		const { task } = query;
+
 		switch ( tab ) {
 			case 'inbox':
 				return <InboxPanel />;
 			case 'orders':
-				const { hasUnreadOrders } = this.props;
```

Comment:
Not a good idea to declare variables inside case's since they'll clash if you reuse them in separate cases.